docs/trees/guide.md

```diff
@@ -256,14 +256,6 @@ println(
 )
 ```
 
-> It's important to keep in mind that quasiquotes expand at compile-time into
-> the same program as if you had written normal constructors by hand. This means
```

Comment:
you lost `the same program` when copying.

Reply:
fixed! https://github.com/scalameta/scalameta/pull/3572/commits/28f654f220623965e0ebb431c0bd8edc0b3fd3b7